README.md

```diff
@@ -86,6 +86,9 @@ $ awesome_bot README.md --allow 403,429
 
 $ awesome_bot README.md --base-url https://github.com/IDR/idr-notebooks/blob/master/
 # will check relative links using the base url provided
+
+$ awesome_bot README.md --base-url https://username:accesstoken@github.com/Organization/Private-repo/blob/master/
```

Comment:
this example is the confusing part.. it suggests username:password only works with --base-url

hmm